docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.1'
+
+services:
+
+  db:
+    image: mysql:8.3.0
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD}
+      MYSQL_DATABASE: ${DATABASE_NAME}
+      MYSQL_USER: ${DATABASE_USERNAME}
+      MYSQL_PASSWORD: ${DATABASE_PASSWORD}
+    volumes:
+      - db_data:/var/lib/mysql
```

Comment:
Consider the possibility of including an environment variable to specify the local path where the database data will be saved in the host machine.